test/literate/roxy.R

```diff
@@ -74,3 +74,24 @@ NULL
 
 ##'
 ##' ¶
+
+
+### 3 ----------------------------------------------------------------
```

Comment:
This test case could be a bit more minimal, i.e. just

```r
##' ¶Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor `incididunt' ut labore et ``dolore'' magna aliqua.
```

Without any other field so it's clearer we are testing for quotes (and we probably should come up with a way of labelling the test cases).